src/lib/serviceRoutinesV2/getEntityAttributeValue.cpp

```diff
@@ -98,17 +109,24 @@ std::string getEntityAttributeValue
         answer = attribute.pcontextAttribute->compoundValueP->render(ciP, JSON, "");
         if (attribute.pcontextAttribute->compoundValueP->isObject())
         {
-            answer = "{" + answer + "}";
+          TIME_STAT_RENDER_START();
+          answer = "{" + answer + "}";
```

Comment:
Not fully sure but I would say that the `answer = "[" + answer + "]" and `answer = "{" + answer + "}"` parts should be left as the were and the TIME_STAT_RENDER_START/STOP be around the function doing the "hard work" to render, i.e. some lines above:

```
answer = attribute.pcontextAttribute->compoundValueP->render(ciP, JSON, "");
```